src/Kinepolis/AuthenticatedKinepolisClient.php

```diff
@@ -72,6 +72,34 @@ public function getMovieDetail(string $token, int $mid): array
         return $contents['movies'][0];
     }
 
+    public function getTheaters(string $token): array
+    {
+        $request = new Request(
+            'GET',
+            $this->movieApiBaseUrl . 'services/content/1.1/theaters',
+            $this->createHeaders($token)
+        );
+
+        $response = $this->client->sendRequest($request)->getBody()->getContents();
+        $contents = Json::decodeAssociatively($response);
+
+        return $contents['theatres'];
+    }
+
+    public function getPricesForATheater(string $token, string $tid): array
```

Comment:
Can we get a docblock what the structure is of this array, or convert it to a dto?

Reply:
The parsers are used for this conversion. The API client and parsers are used in the service.